protected/views/dataset/view.php

```diff
@@ -51,15 +51,29 @@
                                             <div class="dropdown-box">
                                                 <button id="CiteDataset" class="drop-citation-btn dropdown-toggle" type="button" data-toggle="dropdown">Cite Dataset<span class="caret"></span></button>
                                                 <?php
-                                                $text = file_get_contents('https://data.datacite.org/text/x-bibliography/10.5524/' . $model->identifier);
-                                                $clean_text = strip_tags(preg_replace("/&#?[a-z0-9]+;/i", "", $text));
+                                                $url = 'https://data.datacite.org/text/x-bibliography/10.5524/' . $model->identifier;
+                                                $webClient = new GuzzleHttp\Client();
```

Comment:
If we want to fetch information from an external source such as DataCite - this functionality is related to what the [DownloadService](https://github.com/gigascience/gigadb-website/blob/develop/protected/components/DownloadService.php) component class provides?